packages/dashboard/src/components/pages/platforms/components/preview/preview.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { useStores } from 'Stores';
+import { TStringTranslation } from 'Types'
+
+
+const GetSection: React.FC<IGetSectionType> = ({ availability_text, images }) => {
+    const { config_store } = useStores();
+
+    return (
+        <section className='dw-preview'>
+            <div className='dw-preview__item-wrapper'>
+                {images.map((image, idx) => (
+                    <img key={idx} className='dw-preview__item' src={`${config_store.asset_path}/images/${image}`} />
+                ))}
+
+                <Icon className='dw-preview__icon' icon='IcChevronRightBold' width='16' height='16' />
+            </div>
+            <Text color='less-prominent' size='xs'>{availability_text}</Text>
+        </section>
+    );
+};
+
+type IGetSectionType = {
+    availability_text: TStringTranslation;
+    images: string[];
+}
+
+export default GetSection;
```

Comment:
Should be wrapped in `observer` since it's observing `config_store.asset_path`

Reply:
sure, but, is it fine even though asset path is only used once?

Reply:
I checked `config_store` and it looks like `asset_path` is not `observable`, so I was wrong. In general though, whenever we're accessing a value from a store we should use an `observer` wrapper (regardless of the amount of times it's used).